qcodes/station.py

```diff
@@ -313,6 +333,41 @@ def get_config_file_path(
         with open(path) as f:
             self.load_config(f)
 
+    def load_config_files(self,
+                          *filenames: Sequence[str]
```

Comment:
This is not the correct way to type a *args thing. It should be the type of the content 
```suggestion
                          *filenames: str
```